cockatrice/src/dlg_connect.cpp

```diff
@@ -10,18 +11,25 @@
 #include <QKeyEvent>
 #include <QLabel>
 #include <QMessageBox>
+#include <QNetworkReply>
 #include <QPushButton>
 #include <QRadioButton>
 
 #define PUBLIC_SERVERS_URL "https://github.com/Cockatrice/Cockatrice/wiki/Public-Servers"
+#define PUBLIC_SERVERS_JSON                                                                                            \
+    "https://raw.githubusercontent.com/Cockatrice/cockatrice.github.io/master/public-servers.json"
```

Comment:
Looks like this file needs to be added?

Reply:
Does the file not work for you? The formatting was clang-format

Reply:
D'oh, I was looking in the wrong place. Never mind. :upside_down_face:

Reply:
We can link to https://cockatrice.github.io/public-servers.json instead, too!

Reply:
Right, that url is probably more well suited for access from a lot of users

Reply:
Thanks for that, @tooomm . Addressed!